### ISSUE TITLE
Export reports cron erreur session

### ISSUE THREAD

**Issue by Thomas410gone** [mautic/mautic#13827]
### Mautic Version

5.0.x series

### Way of installing

I downloaded a release from https://www.mautic.org/mautic-releases

### PHP version

8.1.27

### What happened?

On Mautic 5.0.4, after upgrade by CLI from 4.4.0
The cron "mautic:reports:scheduler" to export the reports by mail is in error

`console.CRITICAL: Error thrown while running command "mautic:reports:scheduler". Message: "There is currently no session available." {"exception":"[object] (Symfony\\Component\\HttpFoundation\\Exception\\SessionNotFoundException(code: 0): There is currently no session available. at /xxxxxxxxx/htdocs/vendor/symfony/http-foundation/RequestStack.php:126)","command":"mautic:reports:scheduler","message":"There is currently no session available."} {"hostname":"xxxxxxxx","pid":756240}`

### How can we reproduce this issue?

Step 1: Create a report, add some data
Step 2: In "Schedule" tab: choose email report: yes, add email and choose "now" and save
Step 3: Launch CLI : php bin/console mautic:reports:scheduler 
Step 4: Error in logs

### Relevant log output

```shell
console.CRITICAL: Error thrown while running command "mautic:reports:scheduler". Message: "There is currently no session available." {"exception":"[object] (Symfony\\Component\\HttpFoundation\\Exception\\SessionNotFoundException(code: 0): There is currently no session available. at /xxxxxxxxx/htdocs/vendor/symfony/http-foundation/RequestStack.php:126)","command":"mautic:reports:scheduler","message":"There is currently no session available."} {"hostname":"xxxxxxxx","pid":756240}
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by Thomas410gone** [comment]
This issue is linked to  : #13518

**Comment by dsp76** [comment]
@Thomas410gone is it a duplicate of #13518 ? Then please add any additional comments there and we can close this? Also see, that the issue seems to have been fixed for 5.x

**Comment by Thomas410gone** [comment]
@dsp76 yes, this is a duplicate of #13518 but as the original one is closed, there is a message to create a new one.

I still have the issue on 5.0.4 (latest version) so the issue is not fixed

**Comment by RCheesley** [comment]
Closing this issue as it's resolved already.
